cli/command/service/opts.go

```diff
@@ -318,6 +318,19 @@ func convertNetworks(networks []string) []swarm.NetworkAttachmentConfig {
 	return nets
 }
 
+func convertSecrets(secrets []string) []*swarm.SecretReference {
+	sec := []*swarm.SecretReference{}
+	for _, s := range secrets {
+		sec = append(sec, &swarm.SecretReference{
+			SecretID: s,
+			Mode:     swarm.SecretReferenceFile,
+			Target:   "",
```

Comment:
There seems to be an inconsistency between `service create` and `service update`. Shouldn't they both take secrets in the format that specifies a name and target (i.e. use `parseSecrets`)?